andela/blade-runner-cfh
# #153741758 landing page Users should see a landing page with effective UI/UX

### What does this PR do?

- This PR allows a user to see the landing page on visiting the site
- Helps a user get a brief info about the game

### Description of Task to be completed?

N/A

### How should this be manually tested?

- Run npm start to start the server
- Go to localhost:3000 to see the landing page



<img width="1440" alt="screen shot 2018-01-08 at 4 33 51 pm" src="https://user-images.githubusercontent.com/25517388/34715498-2ebd416c-f52d-11e7-9fe5-8951c3aaea72.png">



### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

- Users should see a landing page with effective UI/UX #153741758

### Screenshots (if appropriate)



### 
Questions:

N/A
